api/api/util.py

```diff
@@ -294,3 +294,26 @@ def raise_if_exc(obj):
         _create_problem(obj)
     else:
         return obj
+
+
+from aiohttp import web
```

Comment:
Move import to top of the file and optimize imports

Reply:
The import is deprecated. I have removed it